src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -181,13 +213,13 @@ void EnIk_SetupAction(EnIk* this, EnIkActionFunc actionFunc) {
     this->actionFunc = actionFunc;
 }
 
-void func_80A74398(Actor* thisx, GlobalContext* globalCtx) {
+void EnIk_SetAndInitColliders(Actor* thisx, GlobalContext* globalCtx) {
```

Comment:
On second look this actually does way more than colliders, it's basically the implementation of the init function?